friends/src/Components/FriendList.js

```diff
@@ -0,0 +1,89 @@
+import React ,{Component} from 'react'; /*Importing React and the component class from React*/
+import axios from 'axios'; /*Importing axios, which lets you make API calls in React*/
+
+class FriendList extends Component{
+    state = {friends:[]}; /*Established state and created an empty array called friends*/
+
+    componentDidMount(){
+        this.getFriends(); /*Running the function getFriends*/
+    }
+
+    getFriends = () => { /*Function that gets list of friends from server*/
+        axios.get('http://localhost:5000/friends').then(response => { /*Getting the json array of friends from the server and saving it to a variable*/
+            console.log(response.data);
```

Comment:
Feel free to remove these `console.log()` if they are not being utilized anymore.